### PR TITLE
My first Pull request

### PR DESCRIPTION
**Pull request by shegy28**
**Add-Homepage branch**

In this project we:

Added index.html file
Added a style sheet
Updated the Readme file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (mckent05) [inline .gitignore]: - According to the project requirements, your `.gitignore` file should contain `node_modules`. See your node modules as a cache for your project's dependencies. Please try to fix this 🙏 
**OPTIONAL** - I advise that you install linters locally on your PC so as to make it easier to detect linter errors. You can follow [these](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) instructions

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @shegy28 , 

# STATUS:  CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- Linter checks are passing correctly ✔️ 
- GitHub flow correctly used 👏 👏 
- Descriptive PR title and summary 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (shegy28) [reply .gitignore]: Thanks for the feedback, i took your advice and initialize the npm first and it generated a node_modules file which the code editor suggested to be moved inside the .gitignore file.

**Review by fobadara — APPROVED** [review]
Hi @shegy28, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**